uptane-standard.md

```diff
@@ -240,28 +330,220 @@ The Time Server SHALL expose a public interface allowing primaries to communicat
 
 ## In-vehicle implementation requirements
 
+An Uptane-compliant ECU SHALL be able to download and verify the time, metadata, and image binaries before installing a new image.
+
+Each ECU in a vehicle receiving over-the-air updates is either a primary or a secondary ECU. A primary ECU collects and delivers to the Director vehicle manifests ({{vehicle_version_manifest}}) containing information about which images have been installed on ECUs in the vehicle. It also downloads and verifies the latest time, metadata, and images for itself and for its secondaries. A secondary ECU downloads and verifies the latest time, metadata, and images for itself from its associated primary ECU. It also sends signed information about its installed images to its associated primary.
+
+All ECUs MUST verify image metadata as specified in {{metadata_verification}} before installing an image or making it available to other ECUs. A primary ECU MUST perform full verification ({{full_verification}}). A secondary ECU SHOULD perform full verification if possible, and MUST perform full verification if it is safety-critical. If it is not safety-critical, it MAY perform partial verification ({{partial_verification}}) instead.
+
+### Build-time prerequisite requirements for ECUs
+
+For an ECU to be capable of receiving Uptane-secured updates, it MUST have the following data provisioned at the time it is manufactured or installed in the vehicle:
+
+1. The latest copy of required Uptane metadata at the time of manufacture or install.
+    * Partial verification ECUs MUST have the root and targets metadata from the director repository.
+    * Full verification ECUs MUST have a complete set of metadata from both repositories (root, targets, snapshot, and timestamp), as well as the repository map file {{TAP-4}}.
+2. The public key(s) of the time server.
+3. An attestation of time downloaded from the time server.
+4. An **ECU key**. This is a private key, unique to the ECU, used to sign ECU version manifests and decrypt images. An ECU key MAY be either a symmetric key or an asymmetric key. If it is an asymmetric key, there MAY be separate keys for encryption and signing. For the purposes of this standard, the set of private keys that an ECU uses is referred to as the ECU key (singular), even if it is actually multiple keys used for different purposes.
```

Comment:
It may make even more sense to move the definition to the Terminology section. Let's get these drafts merged into the main doc, so we can see it all together, and then look at ways to revise/restructure for better clarity.